content/docs/csidriver/installation/helm/powerflex.md

```diff
@@ -114,43 +114,43 @@ kubectl -n kube-system kustomize deploy/kubernetes/snapshot-controller | kubectl
+3. Collect information from the PowerFlex SDC by executing the `get_vxflexos_info.sh` script located in the `scripts` directory. This script shows the _VxFlex OS system ID_ and _MDM IP_ addresses. Make a note of the values for these parameters as they must be entered into `samples/secret.yaml`.
 
-4. Prepare `samples/config.yaml` for driver configuration. The following table lists driver configuration parameters for multiple storage arrays.
+4. Prepare `samples/secret.yaml` for driver configuration. The following table lists driver configuration parameters for multiple storage arrays.
 
     | Parameter | Description                                                  | Required | Default |
     | --------- | ------------------------------------------------------------ | -------- | ------- |
     | username  | Username for accessing PowerFlex system. If authorization is enabled, username will be ignored.                       | true     | -       |
     | password  | Password for accessing PowerFlex system. If authorization is enabled, password will be ignored.                     | true     | -       |
-    | systemID  | System name/ID of PowerFlex system.                           | true     | -       |
+    | systemID  | System name or ID of PowerFlex system.                           | true     | -       |
     | allSystemNames | List of previous names of powerflex array if used for PV create     | false    | -       |
     | endpoint  | REST API gateway HTTPS endpoint/PowerFlex Manager public IP for PowerFlex system. If authorization is enabled, endpoint should be the HTTPS localhost endpoint that the authorization sidecar will listen on          | true     | -       |
     | skipCertificateValidation  | Determines if the driver is going to validate certs while connecting to PowerFlex REST API interface. | true     | true    |
     | isDefault | An array having isDefault=true is for backward compatibility. This parameter should occur once in the list. | false    | false   |
     | mdm       | mdm defines the MDM(s) that SDC should register with on start. This should be a list of MDM IP addresses or hostnames separated by comma. | true     | -       |
 
-    Example: `samples/config.yaml`
+    Example: `samples/secret.yaml`
 
 ```yaml
 - username: "admin"
   password: "Password123"
-  systemID: "ID2"
+  systemID: "2b11bb111111bb1b"
   endpoint: "https://127.0.0.2"
   skipCertificateValidation: true 
   isDefault: true 
   mdm: "10.0.0.3,10.0.0.4"
 ```
  *NOTE: To use multiple arrays, copy and paste section above for each array. Make sure isDefault is set to true for only one array.* 
 
-After editing the file, run the below command to create a secret called `vxflexos-config`:
+After editing the file, run the below command to create a secret called `vxflexos-config`. Note that the `vxflexos` can be any user-defined name, in this example, we assume that the name is 'vxflexos':
```

Comment:
I would change this to match the previous note about namespace:
"Note that the namespace can be any user-defined name, in this example, we assume that the namespace is 'vxflexos'"

Reply:
Would something like this be acceptable?

Note that the secret can be any user-defined name, in this example, we assume that the secret is 'vxflexos-config'

Reply:
Did we test any secret name? 
I think there might be a req. that secret ends in "-config":
https://github.com/dell/csi-powerflex/blob/40725457527c2f9f99f87bcd2c2323effb01252c/helm/csi-vxflexos/templates/controller.yaml#L417

Reply:
Since this is helm, we can look at it as changing release name. 
So maybe in example of csi-install.sh, we can say, "modify release name with --release arg, if arg isn't provided, release will be named vxflexos by default"
and then in lines like this one, we can say "this assumes "vxflexos" is release name, but it can be modified during install (link to csi-install.sh section)

Reply:
^ second Jacob

Reply:
Resolved in commit 7a96e18